a2hosting.php

```diff
@@ -51,7 +76,23 @@ public function __construct()
 
 	public function install()
 	{
-		return parent::install() && $this->registerHook('backOfficeHeader');
+		return parent::install() && 
+			$this->registerHook('backOfficeHeader') &&
+			$this->registerHook('header') &&
+			$this->registerHook('actionAuthentication') &&
+			$this->registerHook('actionCartSave') &&
+			$this->registerHook('displayHeader') &&
```

Comment:
You can register header, or displayHeader, but this is useless to register both.